### ISSUE TITLE
test_lint failed for kubeflow-examples-presubmit

### ISSUE THREAD

**Issue by jinchihe** [kubeflow/examples#506]
test_lint failed for kubeflow-examples-presubmit recent days, seems that's broken?

```
INFO|2019-02-12T03:02:11|util.py:45| Running: pylint --rcfile=/mnt/test-data-volume/kubeflow-examples-presubmit-examples-e2e-492-374df77-1328-3f8e/src/kubeflow/examples/.pylintrc /mnt/test-data-volume/kubeflow-examples-presubmit-examples-e2e-492-374df77-1328-3f8e/src/kubeflow/examples/github_issue_summarization/testing/tfjob_test.py 
cwd=/mnt/test-data-volume/kubeflow-examples-presubmit-examples-e2e-492-374df77-1328-3f8e/src/kubeflow/examples
INFO|2019-02-12T03:02:11|util.py:60| Subprocess output:

INFO|2019-02-12T03:02:11|util.py:71| Using config file /mnt/test-data-volume/kubeflow-examples-presubmit-examples-e2e-492-374df77-1328-3f8e/src/kubeflow/examples/.pylintrc
INFO|2019-02-12T03:02:13|util.py:71| ************* Module tfjob_test
INFO|2019-02-12T03:02:13|util.py:71| E: 31, 0: No name 'tf_job_client' in module 'py' (no-name-in-module)
INFO|2019-02-12T03:02:13|util.py:71| E: 32, 0: No name 'test_runner' in module 'py' (no-name-in-module)
INFO|2019-02-12T03:02:13|util.py:71| 
INFO|2019-02-12T03:02:13|util.py:71| -----------------------------------
INFO|2019-02-12T03:02:13|util.py:71| Your code has been rated at 7.50/10


INFO|2019-02-12T03:02:47|util.py:45| Running: pylint --rcfile=/mnt/test-data-volume/kubeflow-examples-presubmit-examples-e2e-492-374df77-1328-3f8e/src/kubeflow/examples/.pylintrc /mnt/test-data-volume/kubeflow-examples-presubmit-examples-e2e-492-374df77-1328-3f8e/src/kubeflow/examples/mnist/web-ui/mnist_client_test.py 
cwd=/mnt/test-data-volume/kubeflow-examples-presubmit-examples-e2e-492-374df77-1328-3f8e/src/kubeflow/examples
INFO|2019-02-12T03:02:47|util.py:60| Subprocess output:

INFO|2019-02-12T03:02:49|util.py:71| Using config file /mnt/test-data-volume/kubeflow-examples-presubmit-examples-e2e-492-374df77-1328-3f8e/src/kubeflow/examples/.pylintrc
INFO|2019-02-12T03:02:50|util.py:71| ************* Module mnist_client_test
INFO|2019-02-12T03:02:50|util.py:71| E: 23, 0: No name 'test_runner' in module 'py' (no-name-in-module)
INFO|2019-02-12T03:02:50|util.py:71| C: 23, 0: third party import "from py import test_runner" should be placed before "import mnist_client" (wrong-import-order)
INFO|2019-02-12T03:02:50|util.py:71| 
INFO|2019-02-12T03:02:50|util.py:71| -----------------------------------
INFO|2019-02-12T03:02:50|util.py:71| Your code has been rated at 7.39/10
INFO|2019-02-12T03:03:05|util.py:45| Running: pylint --rcfile=/mnt/test-data-volume/kubeflow-examples-presubmit-examples-e2e-492-374df77-1328-3f8e/src/kubeflow/examples/.pylintrc /mnt/test-data-volume/kubeflow-examples-presubmit-examples-e2e-492-374df77-1328-3f8e/src/kubeflow/examples/mnist/testing/tfjob_test.py 
cwd=/mnt/test-data-volume/kubeflow-examples-presubmit-examples-e2e-492-374df77-1328-3f8e/src/kubeflow/examples
INFO|2019-02-12T03:03:05|util.py:60| Subprocess output:

INFO|2019-02-12T03:03:06|util.py:71| Using config file /mnt/test-data-volume/kubeflow-examples-presubmit-examples-e2e-492-374df77-1328-3f8e/src/kubeflow/examples/.pylintrc
INFO|2019-02-12T03:03:07|util.py:71| ************* Module tfjob_test
INFO|2019-02-12T03:03:07|util.py:71| E: 31, 0: No name 'tf_job_client' in module 'py' (no-name-in-module)
INFO|2019-02-12T03:03:07|util.py:71| E: 32, 0: No name 'test_runner' in module 'py' (no-name-in-module)
INFO|2019-02-12T03:03:07|util.py:71| 
INFO|2019-02-12T03:03:07|util.py:71| -----------------------------------
INFO|2019-02-12T03:03:07|util.py:71| Your code has been rated at 7.50/10
INFO|2019-02-12T03:03:07|util.py:71| 
INFO|2019-02-12T03:03:08|util.py:45| Running: pylint --rcfile=/mnt/test-data-volume/kubeflow-examples-presubmit-examples-e2e-492-374df77-1328-3f8e/src/kubeflow/examples/.pylintrc /mnt/test-data-volume/kubeflow-examples-presubmit-examples-e2e-492-374df77-1328-3f8e/src/kubeflow/examples/mnist/testing/deploy_test.py 
cwd=/mnt/test-data-volume/kubeflow-examples-presubmit-examples-e2e-492-374df77-1328-3f8e/src/kubeflow/examples
INFO|2019-02-12T03:03:08|util.py:60| Subprocess output:

INFO|2019-02-12T03:03:08|util.py:71| Using config file /mnt/test-data-volume/kubeflow-examples-presubmit-examples-e2e-492-374df77-1328-3f8e/src/kubeflow/examples/.pylintrc
INFO|2019-02-12T03:03:10|util.py:71| ************* Module deploy_test
INFO|2019-02-12T03:03:10|util.py:71| E: 26, 0: No name 'test_runner' in module 'py' (no-name-in-module)
INFO|2019-02-12T03:03:10|util.py:71| 
INFO|2019-02-12T03:03:10|util.py:71| -----------------------------------
INFO|2019-02-12T03:03:10|util.py:71| Your code has been rated at 8.28/10

ERROR|2019-02-12T03:05:09|test_py_lint.py:75| 4 files had lint errors:
/github_issue_summarization/testing/tfjob_test.py
/mnist/testing/deploy_test.py
/mnist/testing/tfjob_test.py
/mnist/web-ui/mnist_client_test.py

```

but I exected that manully on local, that's OK. That's strange...
```
hejinchideMacBook-Pro:example hejinchi$ pylint --rcfile=examples/.pylintrc examples/github_issue_summarization/testing/tfjob_test.py

------------------------------------
Your code has been rated at 10.00/10

hejinchideMacBook-Pro:example hejinchi$ pylint --rcfile=examples/.pylintrc examples/mnist/web-ui/mnist_client_test.py

------------------------------------
Your code has been rated at 10.00/10

hejinchideMacBook-Pro:example hejinchi$ pylint --rcfile=examples/.pylintrc examples/mnist/testing/deploy_test.py

------------------------------------
Your code has been rated at 10.00/10

hejinchideMacBook-Pro:example hejinchi$ pylint --rcfile=examples/.pylintrc examples/mnist/testing/tfjob_test.py

------------------------------------
Your code has been rated at 10.00/10
```

**Comment by jinchihe** [comment]
Checked, the lint test failed recent days.
https://k8s-testgrid.appspot.com/sig-big-data#kubeflow-examples-presubmit

**Comment by texasmichelle** [comment]
It looks like the lint issues were resolved in #493. Feel free to reopen if there are additional errors.
